src/mlmc/sampler.py

```diff
@@ -241,3 +262,21 @@ def l_scheduled_samples(self):
         :return: list
         """
         return self._n_target_samples
+
+    def resurrect_failed(self):
+        """
+        Resurrect failed samples
```

Comment:
Ressurect is possibly not the best name as the new samples are scheduled with the same IDs. I can not recall: Why we need to use the same IDs? Using new IDs we can eliminate modification of HDF data, which is preferred.